tests/test_colour.py

```diff
@@ -62,8 +64,85 @@ def test_from_str(value, expected):
         'rgb(30, -1, 60)',
         'invalid(a, b, c)',
         'rgb(',
+        '#1000000',
+        '#FFFFFFF',
+        "rgb(101%, 50%, 50%)",
+        "rgb(50%, -10%, 50%)",
+        "rgb(50%, 50%, 150%)",
+        "rgb(256, 100, 100)",
     ],
 )
 def test_from_str_failures(value):
     with pytest.raises(ValueError):
         discord.Colour.from_str(value)
+
+
+@pytest.mark.parametrize(
+    ('value', 'expected'),
+    [
+        (discord.Colour.default(), 0x000000),
+        (discord.Colour.teal(), 0x1ABC9C),
+        (discord.Colour.dark_teal(), 0x11806A),
+        (discord.Colour.brand_green(), 0x57F287),
+        (discord.Colour.green(), 0x2ECC71),
+        (discord.Colour.dark_green(), 0x1F8B4C),
+        (discord.Colour.blue(), 0x3498DB),
+        (discord.Colour.dark_blue(), 0x206694),
+        (discord.Colour.purple(), 0x9B59B6),
+        (discord.Colour.dark_purple(), 0x71368A),
+        (discord.Colour.magenta(), 0xE91E63),
+        (discord.Colour.dark_magenta(), 0xAD1457),
+        (discord.Colour.gold(), 0xF1C40F),
+        (discord.Colour.dark_gold(), 0xC27C0E),
+        (discord.Colour.orange(), 0xE67E22),
+        (discord.Colour.dark_orange(), 0xA84300),
+        (discord.Colour.brand_red(), 0xED4245),
+        (discord.Colour.red(), 0xE74C3C),
+        (discord.Colour.dark_red(), 0x992D22),
+        (discord.Colour.lighter_grey(), 0x95A5A6),
+        (discord.Colour.dark_grey(), 0x607D8B),
+        (discord.Colour.light_grey(), 0x979C9F),
+        (discord.Colour.darker_grey(), 0x546E7A),
+        (discord.Colour.og_blurple(), 0x7289DA),
+        (discord.Colour.blurple(), 0x5865F2),
+        (discord.Colour.greyple(), 0x99AAB5),
+        (discord.Colour.dark_theme(), 0x313338),
+        (discord.Colour.fuchsia(), 0xEB459E),
+        (discord.Colour.yellow(), 0xFEE75C),
+        (discord.Colour.dark_embed(), 0x2B2D31),
+        (discord.Colour.light_embed(), 0xEEEFF1),
+        (discord.Colour.pink(), 0xEB459F),
+    ],
+)
+def test_static_colours(value, expected):
```

Comment:
I just want to make a note that some of these have changed before, e.g. `blurple`. I'm not sure what the API stability on these are but we have changed them where it makes sense historically. This has no bearing on the tests being here but I just wanted to comment on it.